src/canvas/Canvas.ts

```diff
@@ -245,8 +245,8 @@ export class Canvas extends SelectableCanvas implements CanvasOptions {
     const shared = {
       e,
       isClick: false,
-      pointer: this.getPointer(e),
-      absolutePointer: this.getPointer(e, true),
+      pointer: this.getHTMLPointFromEvent(e),
+      absolutePointer: this.getCanvasPointFromEvent(e),
```

Comment:
getCanvasPoint won't work because Canvas is also an html term.
Also HTML doesn't really say the whole story because is still from the top-left corner of the canvas and not the screen for example.

I usually call them viewportPoint and designPoint but i want to hear more opinions

Reply:
viewport is risky because the page has a viewport also
I agree the names must be very acurate.

Reply:
maybe getHTMLCanvasRelativePoint(e) and getDesignCanvasRelativePoint, or for this second one, getDocument, getPage CanvasRelativePoint.